subjects/devops/orchestrator/audit/README.md

```diff
@@ -0,0 +1,232 @@
+
+```console
+.
+├── Manifests
+│   └── [...]
+├── Scripts
+│   └── [...]
+├── Dockerfiles
+│   └── [...]
+└── Vagrantfile
+```
+
+###### Are all the required files present?
+
+##### Ask the following questions to the group or student
+
+- What are Container orchestration and what are its benefits?
+
+- What is Kubernetes and what is its main role of it?
+
+- What are K3s and what is their main role?
+
+###### Did the student reply correctly to the questions?
+
+##### Check the Student Documentation
+
+The Documentation must exist in the `README.md` file!
+
+###### Did the README.md file contains the all required information about the solution(Prerequisites, Configuration, Setup, Usage, ...)?
+
+##### Check the docker images in the docker hub:
+
+###### Do the used docker images in the YAML manifest uploaded in the student docker hub account?
+
+##### Check the cluster:
+
+By using k3s in Vagrant 2 virtual machines must be created:
+
+1. Master: the master in the k3s cluster.
+
+2. Agent: an agent in the k3s cluster.
+
+`kubectl` must be installed and configured in the learner machine to manage the cluster.
+
+The nodes must be connected and available, try:
+
+```console
+$> kubectl get nodes -A
+NAME                                           STATUS   ROLES    AGE    VERSION
+<master-node>   Ready    <none>   XdXh   vX
+<agent1-node>   Ready    <none>   XdXh   vX
+$>
+```
+
+###### Does the cluster created by a Vagrantfile?
+
+###### Does the cluster contains 2 nodes (master and agent)?
+
+###### Does kubectl installed and configured in the learner's machine
+
+###### Do the nodes are connected and ready for usage?
+
+##### Check the student infrastructure:
+
+The student must implement this architecture:
+![architecture](../pictures/Architecture.png)
+
+##### Run the student infrastructure:
+
+```console
+user:~$ ./orchestrator.sh create
+cluster created
+user:~$
+```
+
+###### did the student provide an `orchestrator.sh` script that runs and creates and manages the infrastructure
+
+###### did the student respect the architecture?
+
+###### did the infrastructure start correctly?
+
+##### Verify the K8s Manifests:
+
+###### Is there a YAML Manifest for each service?
+
+###### Are credentials not existing in the YAML manifests, except the secret manifests?
+
+##### Ask the following questions to the group or student
+
+- What is infrastructure as code and what is the advantages of it?
+
+- Explain What is a K8s manifest.
+
+- Explain the K8s manifests.
+
+###### Did the student reply correctly to the questions?
+
+##### Check the secrets
+
+```console
+$> kubectl get secrets -o json
+<...>
+$>
+```
+
+###### Are the all used credentials and passwords presented in the secrets?
+
+##### Check all deployed resources:
+
+```console
+user:~$ kubectl get all
+<...>
+user:~$
+```
+
+- `inventory-database container` is a PostgreSQL database server that contains your inventory database, it must be accessible via port `5432`.
```

Comment:
This part has the same improvements already pinned in the subject part. Like Node.js, etc.

Reply:
It's fine if we add it so we can give the student more information about the deployed services.

Reply:
I meant this part has the same issues that were present in the subject:
Like `Node.js` with wrong capitalization and so on, please double-check this section or you could copy again the subject part which is now corrected

Reply:
fixed